docs/core/deploying/deploy-with-cli.md

```diff
@@ -1,201 +1,154 @@
 ---
-title: Deploy .NET Core apps with command-line interface (CLI) tools
-description: Learn to deploy a .NET Core app with command-line interface (CLI) tools
-author: rpetrusha
-ms.author: ronpet
-ms.date: 09/05/2018
+title: Publish .NET Core apps with the CLI
+description: Learn to publish a .NET Core app with the .NET Core SDK command-line interface (CLI) tools.
+author: thraka
+ms.author: adegeo
+ms.date: 01/16/2019
 dev_langs: 
   - "csharp"
   - "vb"
 ms.custom: seodec18
 ---
-# Deploy .NET Core apps with command-line interface (CLI) tools
+# Publish .NET Core apps with the CLI
 
-You can deploy a .NET Core application either as a *framework-dependent deployment*, which includes your application binaries but depends on the presence of .NET Core on the target system, or as a *self-contained deployment*, which includes both your application and the .NET Core binaries. For an overview, see [.NET Core Application Deployment](index.md).
+This article demonstrates how you can publish your .NET Core application from the command line. .NET Core provides three ways to publish your applications. *Framework-dependent deployment* produces a .dll file that uses the locally installed .NET Core runtime. *Framework-dependent executable* produces a platform-specific executable that uses the locally installed .NET Core runtime. *Self-contained* produces a platform-specific executable and includes a local copy of the .NET Core runtime.
 
-The following sections show how to use [.NET Core command-line interface tools](../tools/index.md) to create the following kinds of deployments:
+For an overview of these deployment modes, see [.NET Core Application Deployment](index.md). 
 
-- Framework-dependent deployment
-- Framework-dependent deployment with third-party dependencies
-- Self-contained deployment
-- Self-contained deployment with third-party dependencies
+Looking for some quick help on using the CLI? The following table shows some examples of how to publish your app.
 
```

Comment:
It's listed here because without it, you may not get the FDE. for example, if you're working on a project created when 2.1 was the latest, then you install 3.0 or 2.2 sdks, and you try the command lie, you can't get an FDE for 2.1 but the command works OK. So in general, I tried to stay away from explaining TOO much about the XML. There are so many ways to generate what you want, I figured that this is about the CLI, so lets just give them the 1 command that works. If they want to investigate how to do it differently, they can, but I don't want to make the article too complex.